### PR TITLE
fix mgf

### PR DESCRIPTION
**Pull request by lonelu**
I got a .mgf file from a user. The version 308 could load it, but the current version could not.
It turned out that some mgf files contain empty lines. This pull request skip empty lines in mgf reader.

### CODE REVIEW

**Comment by zrolfs** [comment]
needs unit test

**Comment by lonelu** [comment]
Unit test added. Please merge this pull request when available.

File: Mgf/Mgf.cs [hunk 1]
```diff
@@ -78,6 +84,11 @@ public override MsDataScan GetOneBasedScan(int scanNumber)
                 string line = sr.ReadLine();
                 string[] sArray = line.Split('=');
 
+                if (line == "")
```
Comment (rmillikin) [inline Mgf/Mgf.cs:87]: use `string.IsNullOrWhitespace`

**Review by rmillikin — CHANGES_REQUESTED** [review]

File: Mgf/Mgf.cs (hunk 1 above)
Comment (lonelu) [reply Mgf/Mgf.cs]: Good suggestion!

**Review by rmillikin — APPROVED** [review]

**Review by rmillikin — APPROVED** [review]

**Pull request merged**